eurosurveillance.csl

```diff
@@ -0,0 +1,270 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" et-al-min="7" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en">
+  <info>
+    <title>Eurosurveillance</title>
+    <id>http://www.zotero.org/styles/eurosurveillance</id>
+    <link href="http://www.zotero.org/styles/eurosurveillance" rel="self"/>
+    <link href="http://www.zotero.org/styles/springer-vancouver-brackets" rel="template"/>
+    <link href="https://www.eurosurveillance.org/for-authors#References" rel="documentation"/>
+    <author>
+      <name>Vincent Auvigne</name>
+      <email>vincent.auvigne@ekipaj.com</email>
+    </author>
+    <contributor>
```

Comment:
You can delete Charles from the contributors. 
Contributors are acknowledged through the template link in line 7.